src/UI/Pages/Resources/Page.tsx

```diff
@@ -43,28 +43,32 @@ export const Page: React.FC = () => {
     pageSize,
   });
 
+  const [staleData, setStaleData] = useState(data);
+
+  useEffect(() => {
+    if (RemoteData.isLoading(data)) return;
+    setStaleData(data);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [data.kind]);
```

Comment:
Could you add a test case, that verifies that if the summary data is updated, the summary widget is still correctly updated? 
(So the data is not stale all the time)

Reply:
👍